users/models.py

```diff
@@ -7,7 +7,7 @@ class User(AbstractUser):
     email = models.EmailField(max_length=255, unique=True)
 
     USERNAME_FIELD = "email"
-    REQUIRED_FIELDS = []
+    REQUIRED_FIELDS = ["username"]
```

Comment:
do we need this?

Reply:
I was getting this error previously "TypeError: create_superuser() missing 1 required positional argument: 'username'".
I can have either created the create_superuser method in UserManager that it doesn't require a username field. Alternatively, I added username to the required_fields in my user model. This lets me enter value for username when creating superuser.

Reply:
you need to create model manager